src/app/system/audit-trails/view-audit/view-audit.component.html

```diff
@@ -0,0 +1,120 @@
+<div class="container" fxLayout="row" fxLayoutGap="2%" fxLayout.lt-md="column">
```

Comment:
```suggestion
<div class="container" fxLayout="row"  fxLayoutAlign="space-between" fxLayoutGap.lt-md="4%" fxLayout.lt-md="column">
```
To allow the coverage of 100% of the row, we can try this.